### ISSUE TITLE
Icon size changing for BSBA Learning Resources Pages

### ISSUE THREAD

**Issue by ejpeloza** [oel-mediateam/droplets#28]

**Comment by Lin87** [comment]
Hello, would you happen to have an example on D2L? If yes, could you tell me where to find it. Thanks!

**Comment by ejpeloza** [comment]
When we use multiple paragraphs within a learning resources card for the BSBA program, every additional icon (we are using the external link icon) after the first icon is smaller. The first icon appears to be 5 cm tall, and everyone after that is 4 cm.

For a current work-around, I found that if I put every resource into one paragraph, and used a <br /> tag, the icons size would stay the same. It did not work if I used a <hr /> tag within the same paragraph. 

I have attached an example code selection to display the differences. The second part is the multiple paragraph format that many designers are using, the first card is an example of the work-around. Let me know if you have any questions.

**Comment by ejpeloza** [comment]
looks like it actually put the hr and br code lines into the message. It is suppose to say: For a current work-around, I found that if I put every resource into one paragraph, and used a "br" tag, the icons size would stay the same. It did not work if I used a "hr" tag within the same paragraph.

Also let me know if you can't view the attached file!

Thanks!
Eric

**Comment by ejpeloza** [comment]
And I forgot to put in the live example, sorry for the multiple messages.

For a live course example: UWEX_BSBA_OM300_1 - Evaluating Competitive Priorities in Operations Management. Go to TOC > Project: Correcting Performance Problems in Operations > Learning Resources

**Comment by Lin87** [comment]
Sorry, I don't see the attached file. UWEX_BSBA_OM300_1 is on D2L, right? I can't see it, and searching returned no results. You can drop the file in the comment box.

**Comment by Lin87** [comment]
Yeah, don't paste code directly into comment box. It is using markdown language which also render HTML. Using the Insert Code button to display code.

**Comment by ejpeloza** [comment]
[BSBA_Icon_Size_Bug.txt](https://github.com/oel-mediateam/droplets/files/556859/BSBA_Icon_Size_Bug.txt)
I didn't realize it didn't accept the html file, put it into a txt file. Let me know if it still doesn't work

**Comment by ejpeloza** [comment]
Yeah, the course is in D2L, it comes up when I search for OM 300. Though, you might have to view it while impersonating BSBA faculty role. The bsba courses have been recently added into D2L, and are not automatically added to specific users accounts.

**Comment by Lin87** [comment]
Thanks! Found the issue and will push out a update sometime next week. Just curious, would you happen to know why the page is using [callouts](https://media.uwex.edu/app/droplets/blockcallouts.html) for learning resources instead of a [Droplets Learning Resource](https://media.uwex.edu/app/droplets/learning_resources.html). Totally understand if you don't.

**Comment by ejpeloza** [comment]
Hey Ethan,

Thanks for the update, I’m looping in Eileen regarding your question about the callout use. I’m thinking it was BSBA ID group standards decision, but not sure of why.

Best,
Eric

**Comment by ejpeloza** [comment]
Hi Ethan,
This is a good question.  We deliberated considerably over whether or not to use the learning resources style from Droplets for the business degree.  Ultimately we decided to go the route we did because with the callouts we were able to strike a good balance between consistency and flexibility.
Eileen

From: Peloza, Eric
Sent: Thursday, October 27, 2016 3:17 PM
To: oel-mediateam/droplets <reply@reply.github.com>; oel-mediateam/droplets <droplets@noreply.github.com>
Cc: Author <author@noreply.github.com>; Horn, Eileen <eileen.horn@uwex.edu>
Subject: RE: [oel-mediateam/droplets] Icon size changing for BSBA Learning Resources Pages (#28)


Hey Ethan,

Thanks for the update, I’m looping in Eileen regarding your question about the callout use. I’m thinking it was BSBA ID group standards decision, but not sure of why.

Best,
Eric
